Makefile

```diff
@@ -94,12 +94,32 @@ deps:
 	cargo install --version 1.14.0 hyperfine
 	cargo install --version 0.9.49 cargo-nextest
 	cargo install --version 0.5.9 cargo-llvm-cov
-	pyenv install pypy3.7-7.3.9
-	pyenv global pypy3.7-7.3.9
-	pip install -r requirements.txt
-	pyenv install 3.7.12
-	pyenv global 3.7.12
-	pip install -r requirements.txt
+	cargo install --version 0.11.0 wasm-pack
+	pyenv install pypy3.9-7.3.9
+	PYENV_VERSION=pypy3.9-7.3.9 python -m venv cairo-rs-pypy-env
+	. cairo-rs-pypy-env/bin/activate ; \
+	pip install -r requirements.txt ; \
+	pyenv install 3.9.15
+	PYENV_VERSION=3.9.15 python -m venv cairo-rs-env
+	. cairo-rs-env/bin/activate ; \
+	pip install -r requirements.txt ; \
+
+deps-macos:
```

Comment:
I think this can be simplified to:
```make
deps-macos:
    brew install gmp
    CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib $(MAKE) deps
```

Reply:
I tried with this one and couldn't pass a flag to a make command, plus the pypy installation is different from the `make deps` one.
Aldo, I created a make cargo-deps to make the commands cleaner.